worker-workflow-testing/src/main/java/com/github/cafdataprocessing/workflow/testing/utils/WorkflowHelper.java

```diff
@@ -184,4 +186,47 @@ public static Subdocument createSubdocument(final String reference, final Fields
         task.setDocument(temp);
         return temp;
     }
+
+    /**
+     *
+     * @param id
+     * @param withStringContentPrimaryValue
+     * @param documentBuilders
+     * @return
+     */
+    public static DocumentBuilder buildDocumentBuilder(final String id,
+                                                       final boolean withStringContentPrimaryValue,
+                                                       final DocumentBuilder... documentBuilders)
+    {
+        final DocumentBuilder builder = DocumentBuilder.configure();
+        if (id != null) {
+            builder.withReference(id);
+        }
+        addField("TITLE", "TITLE" + id, true, builder);
+        if (withStringContentPrimaryValue) {
+            addField("CONTENT_PRIMARY", id, true, builder);
+        } else {
+            addField("CONTENT_PRIMARY", id, false, builder);
+        }
+        if (documentBuilders.length > 0) {
+            addField("HAS_ATTACHMENTS", "true", true, builder);
+            builder.withSubDocuments(documentBuilders);
+        }
+        return builder;
+    }
+
+    /**
+     *
+     * @param name
+     * @param value
+     * @param isStringValue
+     * @param builder
+     */
+    public static void addField(final String name, final String value, final boolean isStringValue,
+                                final DocumentBuilder builder)
+    {
+        builder.withFields().addFieldValue(name, value, (isStringValue ? null : DocumentWorkerFieldEncoding.storage_ref))
+            .documentBuilder();
+    }
+
```

Comment:
Are these changes relating to another Jira?  They seem surprising for this Jira.

Reply:
removed them and used a simple documentbuilder.

Reply:
There are now two imports added and some whitespace changes.  Surely all these changes should be removed.